lib/landing_session.js

```diff
@@ -16,31 +15,12 @@ class LandingSession extends Session {
     this.cli = cli;
     this.req = req;
     const { upstream, owner, repo } = this;
-    const upstreamHref = runSync('git', ['config', '--get',
-      `remote.${upstream}.url`]);
-    let warned = false;
-    try {
-      const { hostname, pathname } = new URL(upstreamHref);
-      if (hostname !== 'github.com') {
-        warned = true;
-        cli.warn('Remote repository URL does not point to the expected ' +
-          'host "github.com"');
-      }
-      if (new RegExp(`^/${owner}/${repo}(?:.git)?$`).test(pathname) !== true) {
-        warned = true;
-        cli.warn('Remote repository URL does not point to the expected ' +
-          `repository /${owner}/${repo}`);
-      }
-    } catch (e) {
-      warned = true;
-      cli.warn('Could not parse remote repository URL %j, this might be ' +
-        'because it is not a HTTPS URL', upstreamHref);
-    }
-    if (warned) {
-      cli.warn('You might want to change the remote repository URL with ' +
-        `\`git remote set-url ${
-          upstream
-        } "https://github.com/${owner}/${repo}"\``);
+    const upstreamHref = runSync('git', [
+      'config', '--get',
+      `remote.${upstream}.url`]).trim();
+    if (!upstreamHref.endsWith(`${owner}/${repo}.git`)) {
```

Comment:
Hmm, maybe not check for `.git` in the end, since you can emit adding `.git`?

Reply:
@cPhost Is it possible though? At least the GitHub UI only offers `git@github.com:nodejs/node.git` and `https://github.com/nodejs/node.git` in https://github.com/nodejs/node

Reply:
Yep, when sometimes I am lazy I do `git remote add upstream git@github.com:user/repo` and it works fine, and I just tested if you do `git config --get` it will return URL without `.git` part thus causing a false